.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
🙈

Reply:
Let me guess: You're using VS Code, right? What is this file good for and why do we need it in the project?

Reply:
We don’t, I just use VS Code as an easy way to edit some files like the `project.pbxproj`. I must be accidentally adding it to the commit — it offers no benefit. The commit below updates the `.gitignore` so it will not happen again.